index.md

```diff
@@ -7,86 +7,54 @@ title: MDAnalysis
-updates and events.
-
-## Availability ##
-
-MDAnalysis can be [**easily
-installed**]({{site.baseurl}}/pages/installation_quick_start/) with
-its dependencies using the ``pip`` or ``conda`` package managers;
-MDAnalysis is also available in some recent Linux distributions.
-
-All **source code** is available under the
-[GNU General Public License, version 2](https://www.gnu.org/licenses/gpl-2.0.html)
-(or any later version at your choice) from
-[github.com/MDAnalysis/mdanalysis](https://github.com/MDAnalysis/mdanalysis)
-and the Python Package index
-[pypi.org/project/MDAnalysis](https://pypi.org/project/MDAnalysis).
-
-## Participating ##
-
-Ask **questions** on the [{{ site.mailinglists.discussion.name }}
-forum]({{ site.mailinglists.discussion.url }}) and **be part of the
-conversation**. You can also join the [{{ site.discord.name }}
-Discord Server]({{ site.discord.url }}) to talk with other users and
-developers. (In order to join our Discord server, use the invitation link
-[{{ site.discord.invite }}]({{ site.discord.invite }}).)
-
-Please report **bugs** or **enhancement requests** through the [Issue
-Tracker]({{ site.github.issues }}). 
-
-MDAnalysis is **open source** and welcomes *your* contributions. [Fork
-the repository on
-GitHub](https://github.com/MDAnalysis/mdanalysis#fork-destination-box)
-and submit a pull request. Participate on the [{{
-site.mailinglists.developer.name }}
-forum]({{ site.mailinglists.developer.url }}).
-
-MDAnalysis regularly takes part in various **mentoring and outreach programs**, such as [Google Summer of Code](https://summerofcode.withgoogle.com/), [Google Season of Docs](https://developers.google.com/season-of-docs), [Outreachy](https://www.outreachy.org/), the [Station1 Frontiers Fellowship](https://www.station1.org/sff), and the [CompChemURG](https://www.bindingsites.co.uk/home) mentorship schemes. Many of the [current core developers](/about/#mdanalysis-core-developers/) joined the project through these mentoring schemes. The project also frequently offers **teaching workshops**, both online and in-person; recordings are often made available on the [MDAnalysis YouTube channel](https://www.youtube.com/@mdanalysis3040). Follow our [blog]({{ site.baseurl }}/blog), [LinkedIn](https://www.linkedin.com/company/mdanalysis/), [Twitter](https://twitter.com/mdanalysis), and [Bluesky](https://bsky.app/profile/mdanalysis.bsky.social) pages for updates on how to participate.
-
-
-## Supporting ##
-
-If you like MDAnalysis and want to support our mission, please
-consider making a donation to support our efforts.
+ The **MDAnalysis Project** develops open-source tools for the analysis of molecular simulation data, providing researchers with efficient and accessible solutions for studying molecular structures and dynamics. The project is a fiscally sponsored initiative of [NumFOCUS][], a nonprofit organization that supports open-source scientific computing.
```

Comment:
"initiative" is not the correct wording. You need to use the language from https://numfocus.org/sponsored-projects :

MDAnalysis is a fiscally sponsored project of NumFOCUS, a nonprofit that promotes open practices in research, data, and scientific computing.

(I took the last part from their https://numfocus.org/community/mission page)
```suggestion
The **MDAnalysis Project** develops open-source tools for the analysis of molecular simulation data, providing researchers with efficient and accessible solutions for studying molecular structures and dynamics.
MDAnalysis is a fiscally sponsored project of [NumFOCUS][], a nonprofit that promotes open practices in research, data, and scientific computing.

```
I also inserted a paragraph break.